vault-consul-acl/step1.md

```diff
@@ -126,4 +143,4 @@ Create the policy and token with the `consul acl` command.
   -name consul-servers \
   -rules @server_policy.hcl`{{execute T1}}
 
-This policy will be used to associate the generated Vault tokens.
\ No newline at end of file
+This policy will be used to associate the generated Vault tokens.
```

Comment:
I'd actually move this up to be the second sentence since it's important and is when the users will start to understand how the Vault integration works.